### ISSUE TITLE
[Accessibility]Keyboard focus is invisible after pressing ‘Enter’ key of ToolStrip/StatusStrip with button item

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#3671]
* .NET Core Version: 
.NET SDK 5.0.100-rc.1.20379.18

* Have you experienced this same bug with .NET Framework?: 
Yes

**More Info:**
This issue can also repro on Framework project.

**Minimal repro:**
1.	Extract “ToolStrip_StatusStrip_keyboard_focus.zip” and open it in VS.
[ToolStrip_StatusStrip_keyboard_focus.zip](https://github.com/dotnet/winforms/files/4999180/ToolStrip_StatusStrip_keyboard_focus.zip)
2.	Build and run the project.
3.	Using Tab key to navigate to toolStripButton1 item of ToolStrip, then press Enter key to trigger click event.
4.	Continue to press Enter key and observe the focus of toolStripButton1.

**Expected behavior:**
Keyboard focus should be visible after pressing ‘Enter’ key of ToolStrip/StatusStrip with button item, the behavior should be like button control.

**Actual behavior:**
Keyboard focus is invisible after pressing ‘Enter’ key of ToolStrip with button item.
![ToolStrip_miss_focus](https://user-images.githubusercontent.com/45864017/88887822-be0dce00-d26f-11ea-8508-d55ce7c09163.gif)

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @Marie-Shi. I have investigated this issue and found that this is behavior by design. After clicking on ToolStripButton, ToolStrip return focus to previous control that had focus. Fixing this bug is breaking change and will conflict with expected behavior from #3596 issue. As workaround for this user case we suggest to use next code:
```
private void toolStripButton_Click(object sender, EventArgs e)
{
    ... 
    User's code
    ...
    BeginInvoke(new MethodInvoker(delegate { toolStripButton.Select(); }));
}
```
Call of `BeginInvoke(new MethodInvoker(delegate { toolStripButton.Select(); }));` helps to return focus back to ToolStripButton if it is needed

**Comment by Marie-Shi** [comment]
@SergeySmirnov-Akvelon Yes, I tried this workaround, it works, now Keyboard focus is visible after pressing ‘Enter’ key of ToolStrip with button item. So will you close this GitHub issue?

**Comment by Marie-Shi** [comment]
@Tanya-Solyanik As @SergeySmirnov-Akvelon said this behavior is by design and also provide a workaround. So close it.
